### ISSUE TITLE
[Warlock] Demonic Sacrifice abuse.

### ISSUE THREAD

**Issue by Cabro** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#5307]
When you respec, you should lose all buffs that you gain from talents. Remember Combustion abuse from mages?
Demonic Sacrifice should be removed when you respec our of that talent.
Currently it is not removed, and you can sacrifice Succubus for the buff, and then you can spec into something else while keeping DS up.

This is the same type of bug abuse/exploit as mage's Combustion respec.
Please fix.

**Comment by bogt** [comment]
Intended? Probably not, but maybe it was still possible on retail. The problem is that it's hard to find examples of retail where it was attempted because it was actually expensive to respec then as opposed to the fun-server Warsong, and there was also a better exploit for damage at the time. Some people have been adamant to me that it was possible on retail and we dug up an old forumpost about it (which is annoyingly no longer in google's cache anymore, can't find it now unless Salce saved the URL?) but it's also referred to here: http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/2041-warlock-exploit.html
"and you could respec to Shadow Mastery/Ruin, and you keep the Master Demonologist aura and Demon Sacrifice buff untill you logout or die."
Having said that, when was the last time anything was fixed to Feenix 1.12? Whether it's intended or not it gives a competitive edge in PvE (which is already being steamrolled regardless), and it's pretty useless in PvP where you'd get better use out of an actual pet. Leave it as is.

**Comment by Zerixx** [comment]
Yeah, this is because I screwed up the latest build. In the past build, we worked on the spell system, and I mistakenly removed the override functions, thus giving players spells they don't actually have.

This is fixed in the next build. We may not see it today, but we'll see it soon. 

- Z

**Comment by Cabro** [comment]
You'd like that, wouldn't you?

**Comment by bogt** [comment]
Insightful. Sorry, but I'm indifferent to feenix PvE mechanics, I only briefly tried to play that class competitively and got bored when shadowburn GCD was fixed. My warlock is catching dust along with all my other characters. Maybe the stars will align in such a way that your next response could contain some sort of evidence or counter-evidence to this issue but I'm sure it'll just wind up being more Balkan-infused drivel.
